### ISSUE TITLE
Hunter on C++ weekly

### ISSUE THREAD

**Issue by dvirtz** [ruslo/hunter#1457]
Jason Turner has started a mini series on package managers as part of his C++ weekly show and made an episode on Hunter:
https://youtu.be/O2_N8OzPGWQ

I think it turned out pretty well. He has a valid point there about the easiness (or lack of) getting HunterGate. 

What do you think about the argument against using -isystem? Is it configurable?

**Comment by ruslo** [comment]
> What do you think about the argument against using -isystem? Is it configurable?

As one of the commenters notes it's from `fmt` package itself. Hunter do not control such things, it's all CMake standard mechanisms.

> He has a valid point there about the easiness (or lack of) getting HunterGate

Copy part with `wget` added to documentation:
* https://docs.hunter.sh/en/latest/quick-start/boost-components.html

I have cloned Hunter repository with "gate" submodule. Usually I copy it from there.


Just for the reference this video so far the best I saw about starting with Hunter:
* https://www.youtube.com/watch?v=TeeLEfT8nCU

**Comment by Bjoe** [comment]
Cool Jason start a presentation about hunter :-)... hopefully hunter is getting now popular .... for easiness .... I created a shell scritpt that generate a CMakeLists.txt with the latest hunter version, see in my repo [create-cmake-project-with-hunter](https://github.com/Bjoe/create-cmake-project-with-hunter), checkout and execute `./update-hunter.sh` ... at the moment it is not really updating, it is has only support to "get latetst hunter" and create CMakeList.txt. As requirement you need `lynx`.

**Comment by Knitschi** [comment]
Is there any chance that HunterGate.cmake could become an official CMake module? Have you asked the cmake guys if they see a chance for that?

**Comment by ruslo** [comment]
> Have you asked the cmake guys if they see a chance for that?

I have mentioned it once in mailing list but I have never asked. Even if HunterGate is used for a quite long time and do not change often I still feel that it's not stable enough :) That is the main problem for me.

For example:
* https://github.com/hunter-packages/gate/blob/936eab75b46630afdae49fb1d8dac0b3f50c1e4d/cmake/HunterGate.cmake#L63 see https://github.com/ruslo/hunter/issues/677
* https://github.com/hunter-packages/gate/blob/936eab75b46630afdae49fb1d8dac0b3f50c1e4d/cmake/HunterGate.cmake#L120 see https://github.com/ruslo/hunter/issues/558#issuecomment-255492072
* https://github.com/hunter-packages/gate/blob/936eab75b46630afdae49fb1d8dac0b3f50c1e4d/cmake/HunterGate.cmake#L495-L498 I think this feature should be removed
* https://github.com/hunter-packages/gate/blob/936eab75b46630afdae49fb1d8dac0b3f50c1e4d/cmake/HunterGate.cmake#L368-L370 see https://github.com/ruslo/hunter/issues/1341

**Comment by Knitschi** [comment]
Yes, making changes to the file will be much slower then :). But maybe in the future.
